src/algebra/big_operators/order.lean

```diff
@@ -170,23 +170,23 @@ calc f a = ∏ i in {a}, f i : prod_singleton.symm
   prod_le_prod_of_subset_of_one_le' (singleton_subset_iff.2 h) $ λ i hi _, hf i hi
 
 @[to_additive]
```

Comment:
```suggestion
@[to_additive sum_le_nsmul_of_forall_le]
```

Reply:
If I use `npow` instead of `pow` (which is probably correct anyways), wouldn’t `to_additive` then do the right thing due to https://github.com/leanprover-community/mathlib/blob/62e7d353a37364039d9465a7a8cb1a6125eb24ce/src/algebra/group/to_additive.lean#L216 ?

Reply:
Yes, but the naming convention wants `pow` not `npow`

Reply:
Looks like it!

Reply:
Oh, the naming convention doesn’t match the name of the thing?

In that case, how about adding
```
 | is_comm ("pow" :: s)                := add_comm_prefix is_comm "nsmul"     :: tr ff s 
```
to make `to_additive` aware of the naming convention/

Reply:
Or is this too blunt, as the `pow` can refer to other forms of `_ ^ _` and so an automatism isn’t going to be helpful?

Reply:
Looking good in the sense that plenty of existing lemmas now don’t need an explicit name anymore. Worth doing like this? Should I do this in a separate PR?

Reply:
> In that case, how about adding

That might be a good idea, but that definitely pushes this PRfirmly outside of `easy` territory. There are 95 matches for `to_additive[^\]]*nsmul`, so making that automatic is definitely a job for a standalone PR.

Can you just add the three `@[to_additive the_explicit_nsmul_name]` for now, to avoid going further down the rabbit hole?

Reply:
I’m already half way through the rabbit hole, so I’ll extract this into a separate PR :-)

Reply:
This rabbit hole is filled at https://github.com/leanprover-community/mathlib/pull/12477